README.md

```diff
@@ -1,2 +1,99 @@
 # PlotEat
-This is an apprenticeship project
+
+PlotEat is an meal planing shopping list app, available at https://ploteat.herokuapp.com/ . 
+
+## Architecture
+
+This is a Rails applicaction that allows you to store recipes, create a menu with your recipes and create a shopping list from the recipes that are on the menu.
+
+## Development
+
+Instructions for how to set up, work with and deploy the project.
+
+
+### Dependencies
+
+This is a
+*[Ruby](https://www.ruby-lang.org)*
+project. `bin/setup` will guide you through installing the required language(s). If you prefer to do this manually, or
+know that you already have the requirements installed `bin/setup` will allow you to skip the language setup and allow
+you to proceed with installing your language(s) manually as you see fit.
+
+All application dependecies can be installed via `bin/setup`. You will have the option to skip each of the below categories.
+To manually install dependencies you can run *`brew bundle`, `bundle install`, `yarn install`, and/or `mix deps.get`*.
```

Comment:
I think you can remove `yarn install`, or do you have any javascript dependencies?

mix deps.get is for Elixir, so def remove that.